.github/workflows/prbuild.yaml

```diff
@@ -15,8 +15,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.17
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v2
+        uses: golangci/golangci-lint-action@v3
```

Comment:
Updating the linter is still causing issues, we'll have to debug those some more -- maybe save that for its own PR so it doesn't get in the way of the upgrades for the other actions.